plugins/blocklayer/mana_plugin.go

```diff
@@ -249,22 +249,22 @@ func runManaPlugin(_ *node.Plugin) {
 					}
 
 				}
-
-				if err := snapshot.LoadSnapshot(Parameters.Snapshot.File, headerConsumer, utxoStatesConsumer, epochDiffsConsumer); err != nil {
+				// initialize cMana WeightProvider with snapshot
+				activityLogConsumer := func(activityLog epoch.SnapshotEpochActivity) {
```

Comment:
This wrapping function seems to be unnecessary, it could directly be `deps.Tangle.WeightProvider.LoadActiveNodes`